change/@azure-msal-common-76a4fd19-0905-46ee-b183-fae2ad84478f.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
```

```suggestion
  "type": "major",
```

I think for the changes needed for the `BaseClient` constructor, this will need to be a major bump of `msal-common`.